worksheets/kotlin/1-krumble/index.md

```diff
@@ -0,0 +1,187 @@
+---
+layout: page
+title: Introduction to Kotlin by building a cookie themed tinder
+---
+
+The aim of this tutorial is to build a dating like app for cookies! We'll have a list of cookies to either like or dislike!
+
+![Imgur](https://i.imgur.com/3u9ATGZ.png)
+
+Before starting, please ensure you have Android Studio installed and setup correctly.
+
+Also, feel free to pick a different topic than cookies if you prefer. You could pick animals, fruit, or even different little green robots.
+
+
+*****
+
+## 1 Starting a new Android Studio Project
+
+Firstly we want to create a new app!
+
+![Imgur](https://i.imgur.com/7a8hL4R.png)
+
+On the first screen select a good **name** for your app. This is what people will see when they install your app. I went for crumble because it's similar to bumble but cookie themed.
+
+Next we have a field called **namespace** - this is how Google sandboxes your app. You want to ensure this is unique. *Talk to your coach about what sandboxing means if you're not sure.*
+
+Finally on this screen we need to ensure the most important checkbox is ticked for this workshop! **Enable Kotlin support!**
+
+On the proceeding screens, you want to set a Min SDK - this is the minimum version of Android we want to support. You can click on the version number to get more information and see exactly how many people in the world who could install your app!
+
+*Speak to your coach about what version they have to use at work and the fun they have involved with that!*
+
+Next, we want to create an Empty Activity. I'd keep the default names of `MainActivity` and `activity_main`.
+
+Finally you can click finish! This might take some time to actually create everything needed for your shiny new app ✨
+
+*Take some time to look at the different buttons with your coach - where are the different files located in the project? How do you run your app in an emulator or on your device?*
+
+## 2 Layout
+
+If you feel confident with layouts already, or would prefer to focus on purely kotlin this time. Copy paste the code in this file <https://gist.github.com/daniellevass/aa6d4042b950aaf6c0f2345530e4baa4> into your layout. This will give you all the layout elements we'll use later!
```

Comment:
It might be worth making it more obvious that this is an optional step and something they could skip in favour of doing the steps below. Maybe make it italic?